### ISSUE TITLE
Strava Import fails on activities without gps data

### ISSUE THREAD

**Issue by klofrau** [martin-ueding/geo-activity-playground#99]
Some of my activies (e.g. manually added activies) do not contain any gps information, hence no gpx or similar file is provided with the export.
This results in the following error message:

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/none/repos/geo-activity-playground/geo_activity_playground/__main__.py", line 98, in main
    options.func(options)
  File "/home/none/repos/geo-activity-playground/geo_activity_playground/__main__.py", line 62, in <lambda>
    func=lambda options: convert_strava_checkout(
                         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/none/repos/geo-activity-playground/geo_activity_playground/importers/strava_checkout.py", line 214, in convert_strava_checkout
    activity_file = checkout_path / row["Filename"]
                    ~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~
TypeError: unsupported operand type(s) for /: 'PosixPath' and 'float'



My workaround was to delete the corresponding lines from activities.csv. I'm sure there is a better way 😃

**Comment by martin-ueding** [comment]
These activities have no filename, therefore Pandas makes it a NaN, which is a float value. It is easy to detect, we'll just skip these. Will be released as part of 0.20.0. If that doesn't help, please reopen this ticket.

**Comment by martin-ueding** [comment]
Version 0.20.0 is released you can update and try it out. If the issue persists, please reopen this ticket.
